http-add-on/README.md

```diff
@@ -82,6 +82,7 @@ their default values.
 
 | Parameter                                                  | Description                               | Default                                         |
 |:-----------------------------------------------------------|:------------------------------------------|:------------------------------------------------|
+| `additionalLabels`                                         | Additional labels to apply to KEDA workloads | `{}` |
```

Comment:
Let's move it to the end of the list since it's less commonly used

Reply:
Done.